node/src/Cardano/Node/API.hs

```diff
@@ -264,16 +341,14 @@ instance HasLens LastKnownHeaderTag InfoCtx LastKnownHeader where
     lensOf =
         lens infoCtxLastKnownHeader (\i s -> i { infoCtxLastKnownHeader = s })
 
-instance
```

Comment:
The `MonadTrans` instance for `MonadDBRead` is super frustrating. It clashes with any polymorphic implementation like this, and even an `OVERLAPPING` pragma doesn't shut it up. So, you need dozens of instances like this.